Tilganger.md

```diff
@@ -0,0 +1,43 @@
+## Altinn Formidlingstjeneste - servicedetaljer
+<table>
+	<tbody>
+		<tr>
+			<td><p><strong>Navn</strong></p></td>
+			<td><p><strong>ServiceCode</strong></p></td>
+			<td><p><strong>ServiceEditionCode</strong></p></td>
+		</tr>
+		<tr>
+			<td><p>AFPANT (Altinn Test TT02)</p></td>
+			<td><p>4752</p></td>
+			<td><p>1</p></td>
+		</tr>
+		<tr>
+			<td><p>AFPANT (Altinn Prod)</p></td>
+			<td><p>4752</p></td>
+			<td><p>1</p></td>
+		</tr>
+	</tbody>
+</table>
+*NB:* ServiceCode for tjenesten er lik i Test og Prod.
+
+## Maskinell kontroll av Altinn-tilganger (for systemleverandører)
+Det er mulig å foreta en maskinell kontroll av hvilke organisasjoner som har delegert rettigheter til systemleverandør for en gitt ServiceCode. 
+[Les mer om maskinell kontroll her](altinn-maskinell-kontroll.md)
+
+## Altinn-tilgang for systemleverandører/datasentraler
+Kartverket må gi rettigheter (READ+WRITE) i tjenesteeierstyrt rettighetsregister for alle systemleverandører/datasentraler som skal koble seg direkte til denne tjenesten.
+Bestillinger av denne tilgangen må gjøres via Kartverket JIRA (https://jira.statkart.no/).
+
+## Altinn-tilgang for eiendomsmeglerforetak/banker
+Kartverket må gi rettigheter (READ+WRITE) i tjenesteeierstyrt rettighetsregister for alle bank- og eiendomsmeglerforetak som skal sende- og motta forsendelser fra denne tjenesten, selv om forsendelsene skal hentes/administreres av tredjepart (systemleverandør/datasentral).
+Bestillinger av denne tilgangen må gjøres via tredjepartsleverandør eller direkte til Kartverket.
+
+## Delegering av roller fra egne kunder til systemleverandør/datasentral
+Systemleverandører/datasentraler som skal utføre sending/mottak på vegne av *andre organisasjoner* (eks meglerforetak/bank) må registrere seg selv hos Kartverket (ref ovenstående punkt).
+Ved henting av forsendelser på vegne av egne kunder skal systemleverandør/datasentral bruke organisasjonsnummeret tilhørende kunden som "reportee" mot Altinn.
+Systemleverandører/datasentraler må også hente meldinger for sitt *eget organisasjonsnummer* (for det er dit ACK/NACK meldinger fra mottakersystem sendes). 
```

Comment:
Hm..er litt usikker på om denne setningen stemmer atm/lenger? Stemmer det ikke at alle ACK/NACK meldinger sendes med bank/megler og vice versa som Altinn reportee/recipient? Sjekket koden vår nå for sikkerhets skyld og det ser ikke ut som at vi poller for meldinger til vårt eget orgnr/VIsma Real Estate AS.

Reply:
Kan ta opp dette på teknisk møte i morgen.